### PR TITLE
Added stemming in english and french

### PR DESCRIPTION
**Pull request by maxneuvians**
Fix for #652

### CODE REVIEW

File: store.js [hunk 1]
```diff
@@ -29,8 +34,8 @@ export const reducer = (state = initialState, action) => {
     case "INDEX_BENEFITS":
       benefits = state.benefits;
       enIdx = lunr(function() {
-        this.pipeline.remove(lunr.stemmer);
-        this.pipeline.remove(lunr.stopWordFilter);
+        //this.pipeline.remove(lunr.stemmer);
+        //this.pipeline.remove(lunr.stopWordFilter);
```
Comment (sastels) [inline store.js]: 🔥

File: store.js [hunk 2]
```diff
@@ -40,9 +45,10 @@ export const reducer = (state = initialState, action) => {
       });
 
       frIdx = lunr(function() {
-        this.pipeline.remove(lunr.stemmer);
-        this.pipeline.remove(lunr.stopWordFilter);
+        //this.pipeline.remove(lunr.stemmer);
+        //this.pipeline.remove(lunr.stopWordFilter);
```
Comment (sastels) [inline store.js]: 🔥

**Comment by sastels** [comment]
- Is there any way to have "family" highlighted when searching for "families"?
- on main page, typing "benefits" stops showing "caregiver benefit" as an option. Is there any way to do this?
- can we start the search when the user hits the return key in the search bar (main page) rather than having to use the search button (not related to stemming, but it bugs me)

**Review by sastels — APPROVED** [review]
minor questions, awesome work!

**Comment by maxneuvians** [comment]
Made three new issues based up recommendations

**Pull request merged**